### ISSUE TITLE
Upgrading Visual Studio from 17.3 to 17.5 causes Blazor errors in the console

### ISSUE THREAD

**Issue by kakins** [dotnet/runtime#82778]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Similar to this: https://github.com/dotnet/aspnetcore/issues/45271 - I tried updating my packages, didn't help.

![image](https://user-images.githubusercontent.com/14915247/221711341-e016468f-36f9-4907-b086-fe3a5c60dcfd.png)

### Expected Behavior

These errors weren't here prior to upgrading.

### .NET Version

.NET 6

**Comment by mkArtakMSFT** [comment]
@thaystg can you please look into this? Thanks!

**Comment by thaystg** [comment]
@kakins, this is an expected behavior because we implemented new features, you need to update your devserver package in your csprojs to at least version 6.0.10.

`<PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly.DevServer" Version="6.0.10" PrivateAssets="all" />`

**Comment by kakins** [comment]
@thaystg I did update my DevServer package to the latest .NET 6 version and still saw the issue.

**Comment by thaystg** [comment]
> @thaystg I did update my DevServer package to the latest .NET 6 version and still saw the issue.

Did you update it on all the projects of your solution that uses this package?
And also which is the version that you are using?
And I'm sorry to ask, did you rebuild the app?
If you can share your project it will be helpful also.

Thanks.

**Comment by kakins** [comment]
@thaystg Those are all good questions. Let me try that as soon as I can and I'll get back to you.

**Comment by kakins** [comment]
@thaystg Ah, I found the problem.  I updated my packages in the `Client` project...
![image](https://user-images.githubusercontent.com/14915247/222551593-04bb46e5-8c2e-4aef-b4f5-16666eb071f9.png)

...but not the server.  I then updated the `Server` project packages also to `6.0.14` and the error disappeared.  
![image](https://user-images.githubusercontent.com/14915247/222551885-6bfbf0ec-ce19-479b-b727-4cda59dcdbd9.png)
 
Thank you for your patience and help!
